recipe/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd src
+
+TCL_VERSION="$(echo 'puts [info tclversion]' | tclsh)"
+TCL_LIB_PATH="$PREFIX"/lib/tcl"$TCL_VERSION"
+BWIDGET_PATH="$TCL_LIB_PATH"/bwidget
+
+[[ ! -d "$BWIDGET_PATH" ]] || exit 1
```

Comment:
This logic makes me mentally stumble.  Is there a logical difference with

```suggestion
[[ -d "$BWIDGET_PATH" ]] && exit 1
```

I assume the test exists because we want to panic if `bwidget` already exists.